test/performance/commCompare/gbt/many-to-one.ml-compopts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+# Aim for about 1 second of run time.  The number of operations needed
```

Comment:
Personally, I'd be in favor of run times between 5-10 seconds

Reply:
I'll adjust to aim for about 5 secs.